### DISCUSSION TITLE
Welcome to PCT discussions!

### DISCUSSION THREAD

**Discussion by gquerret** [Riverside-Software/pct#438] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nvanelck** [comment]
My name is Niek van Elck, from Utrecht, the Netherlands. Not the brightest of the bunch but definitely a Progress and PCT enthousiast for over 20 years now. 
I'm working with Progress, Jenkins, PCT and Sonarqube for a couple of years now but I know I'm not getting everything out of it what I should so hopefully I can post questions and maybe share ideas. 
Regards, 

Niek

**Comment by cverbiest** [comment]
Hi, I'm Carl Verbiest . I work at CCE where I'm responsible for our internal development framework. Started using Progress at version 6.2, PCT user and occassional contributor.
Experience with OpenEdge, Linux, Nagios, Jenkins, Sonarqube for Openedge. 
Regular attendee of PugChallenge.
Google, Progress communities and Stackexchange help me as I try to avoid reinventing the wheel.

During lockdown xmas holiday I started exploring docker, portainer, mqtt, nodered on a raspberry pi.

**Comment by gquerret** [comment]
And I didn't introduce myself ! Gilles Querret, from Lyon in France. I started working on PCT in 2003, as the build process in my company was really flawed. As I strongly believe in open-source, I wanted to share this tool and tried to make it useful for the entire community. Also sharing the core parts of the OpenEdge plugin for SonarQube, though rules are commercial (Riverside Software has to pay the bills). My idea behind opening this GitHub discussion space was to get some feedback about open-source in the OpenEdge community. But any other topic will be welcome !

@cverbiest Raspberry Pi are great, hope you have fun with them !

**Comment by Tranborg** [comment]
Hi! For a while ago PCTCompiler has added the logfile with this...

```
  [PCTCompile] Total time taken to index: 585
  [PCTCompile] Total number of files indexed/removed : 9
  [PCTCompile] Total time taken to index: 591
  [PCTCompile] Total number of files indexed/removed : 1
  [PCTCompile] Total time taken to index: 567
  [PCTCompile] Total number of files indexed/removed : 13
  [PCTCompile] Total time taken to index: 1056
  [PCTCompile] Total number of files indexed/removed : 13
  [PCTCompile] Total time taken to index: 724
  [PCTCompile] Total number of files indexed/removed : 14
  [PCTCompile] Total time taken to index: 663
  [PCTCompile] Total number of files indexed/removed : 1
  [PCTCompile] Total time taken to index: 572
  [PCTCompile] Total number of files indexed/removed : 13
  [PCTCompile] Total time taken to index: 763
  [PCTCompile] Total number of files indexed/removed : 1
  [PCTCompile] Total time taken to index: 611
  [PCTCompile] Total number of files indexed/removed : 10
...
```

Q: Is it possible to switch this off, but still keep the Total files compiled in the log?

BR
Lennart Tranborg

**Comment by gquerret** [comment]
@Tranborg  Which version of Progress do you use ?

**Reply by Tranborg** [reply]
12.3.0.00

**Reply by gquerret** [reply]
@Tranborg There's no such message in PCTCompile. Do you use a custom version of PCT ? Or the Gradle plugin ?

**Reply by Tranborg** [reply]
Not sure. The standard from Progress i guess.

btw here are the ANT code I use.

    
    <target name="compile"
            description="Compile project">

        <echo>Deleting and then building into ${build.dir}</echo>

        <delete dir="${build.dir}" />
        <delete>
            <fileset dir="${src.dir}">
                <include name="**/*.r" />
                <exclude name="" />
            </fileset>
        </delete>


        <PCTCompile dlcHome="${DLC}"
                    destdir="${build.dir}"
                    listing="true"
                    xmlXref="true"
                    keepXref="true"
                    relativePaths="true"
                    inputChars="32000"
                    tempDir="${tmp.dir}"
                    assemblies="${project.dir}"
                    forceCompile="yes">
            <fileset dir="${src.dir}" includes="**/*.p,**/*.cls" />
            <propath>
                <pathelement path="${src.dir}" />
                <pathelement path="${DLC}/tty/netlib/OpenEdge.Net.pl" />
            </propath>
            <DBConnection dbName="${db.name}"
                          singleUser="${db.single.user}"
                          hostName="${db.host}"
                          logicalName="${db.ldb.name}"
                          dbPort="${db.port}"
                          dbDir="${db.dir}" />
        </PCTCompile>

        

    </target>




and here are the pct dir of DLC. 
PCT.jar date 2020-10-06, 3113 KB

![image](https://user-images.githubusercontent.com/42405751/182100512-41f3ef92-746f-49dc-b3af-7a70d68b0291.png)

**Comment by Tranborg** [comment]
.pct files will make Progress Catalog generator crash. This issue is also sent to Progress support today, but I just want to fish here also.

The effect is if PCT Compiler is used in the project. Target is ./tmp/build
The Edit ABL Service, Create a Data Object service, ERROR! At lease one resource must be selected.
If the temp files in .pct is deleted the Catalog generator works fine again.

Have a good day, and thanx for this forum I now will try use more.

Br /Lennart Tranborg, CGI Sweden.
